### PR TITLE
Fix(MAF): Run MIP-DNA for wgs fastq samples

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
## Description
*This PR fixes a bug that stopped fastqs from being sent to MIP for genotyping*

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test submission
- [ ] do submit a fastq order with wgs samples 

### Expected test outcome
- [ ] check that the cases get data_analysis equal to mip-dna

### How to test fixing of old samples
- [ ] find a case with data_analysis == fastq,  data_delivery == fastq and application with prep_category == "wgs", prio research and created after 2021-02-01
- [ ] run alembic upgrade on stage  

### Expected test outcome
- [ ] check that the cases get data_analysis equal to mip-dna

- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in changelog
- [ ] Deploy this branch
- [ ] Inform to production

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result: 
input:
image
wgs?
image
Result:
image
🆗 MIP-DNA for wgs

File: cg/meta/orders/status.py [hunk 1]
```diff
@@ -391,6 +394,14 @@ def store_fastq_samples(
         self.status.add_commit(new_samples)
         return new_samples
 
+    @staticmethod
+    def if_type_is_wgs_then_we_should_do_mip_dna_analysis_for_maf(application_version):
```
Comment (moonso) [inline cg/meta/orders/status.py:398]: longest name in cg 😄

Comment (moonso) [reply cg/meta/orders/status.py:398]: Could you type hint input and output?

**Review by moonso — APPROVED** [review]
👍

File: cg/meta/orders/status.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg/meta/orders/status.py:398]: Would not: `if_type_wgs_do_mip_dna_analysis_for_maf` suffice? 😄

Comment (patrikgrenfeldt) [reply cg/meta/orders/status.py:398]: good suggestion, fixed

**Comment by patrikgrenfeldt** [comment]
Test result db-migration:
image
🆗 cases that are fastq and wgs were set as MIP-DNA

**Comment by patrikgrenfeldt** [comment]
Test result db-upgrade:
image
🆗 changed data-analysis to mip for fastq wgs cases

**Comment by patrikgrenfeldt** [comment]
Deployed on Hasta:
image

image


Deployed on clinical-db:
image

**Pull request merged**